CONTRIBUTING.md

```diff
@@ -101,16 +101,13 @@ We follow [Open source participation guidelines](https://www.redhat.com/en/resou
 
 ### Coding Guidelines
 
-We use the maven `checkstyle` and `sortpom` plugins which are skipped by default for older modules
-and enabled for newer ones.
+We use the maven `checkstyle` and `sortpom` plugins which are enabled by default, but skipped for older modules.
 
 For older modules where checkstyle is disabled, when editing a file you should either
-a) follow the style currently used in the file you have edited or,
-b) reformat it according to our checkstyle ruleset (by temporarily enabling checkstyle
-on the affected module) but separate functional changes from formatting changes into different commits
-in the same PR (this rule facilitates traceability).
-In other words a file (in a module with checkstyle disabled) may only be re-formatted if it also
-contains a functional change and is made in a separate commit.
+a) reformat it according to our checkstyle ruleset (by temporarily enabling checkstyle on the affected module) but separate functional changes from formatting changes into different commits in the same PR (this rule facilitates traceability).
```

Comment:
@marcosgopen I think this part of the changeset should be reverted as I don't think we should be altering the ordering of the options. Please can I ask if you are OK to revert this part of the change?

Reply:
mmh, I am ok with your review. But could I ask you to read the https://issues.redhat.com/browse/JBTM-3302 description and tell me if you still have the same opinion as my change is aligned to that issue.

Reply:
However I think you are right and this is not the right place where to change it, so I have just removed it as you asked and updated the PR.